From ea8d34499abe54c34adaec344e16db7efeb4400d Mon Sep 17 00:00:00 2001
From: Simon Pintarelli <simon.pintarelli@cscs.ch>
Date: Sat, 9 Dec 2023 17:01:23 +0100
Subject: [PATCH] osx fix cmake

---
 CMakeLists.txt | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 352edd53c..7feefb5ce 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -149,11 +149,11 @@ endif()
 find_package(Filesystem COMPONENTS Final Experimental)
 
 add_library(sirius::filesystem INTERFACE IMPORTED)
-  target_link_libraries(sirius::filesystem INTERFACE std::filesystem)
-  target_compile_definitions(sirius::filesystem INTERFACE
-    $<$<BOOL:${Filesystem_FOUND}>:SIRIUS_STD_FILESYSTEM>
-    $<$<BOOL:${CXX_FILESYSTEM_IS_EXPERIMENTAL}>:SIRIUS_STD_FILESYSTEM_EXPERIMENTAL>
-  )
+# target_link_libraries(sirius::filesystem INTERFACE std::filesystem)
+# target_compile_definitions(sirius::filesystem INTERFACE
+#   $<$<BOOL:${Filesystem_FOUND}>:SIRIUS_STD_FILESYSTEM>
+#   $<$<BOOL:${CXX_FILESYSTEM_IS_EXPERIMENTAL}>:SIRIUS_STD_FILESYSTEM_EXPERIMENTAL>
+# )
 
 if (SIRIUS_USE_OPENMP)
   find_package(OpenMP REQUIRED)
-- 
2.43.0

